tests/test_perseptual.py

```diff
@@ -0,0 +1,55 @@
+import torch
+import pytest
+from typing import Any, List
+
+from photosynthesis_metrics import ContentLoss, StyleLoss, LPIPS
+
+
+@pytest.fixture(scope='module')
+def prediction() -> torch.Tensor:
+    return torch.rand(4, 3, 128, 128)
+
+
+@pytest.fixture(scope='module')
+def target() -> torch.Tensor:
+    return torch.rand(4, 3, 128, 128)
```

Comment:
It's time to extend tests. Do not test your code on only one ideal input data option

Reply:
Added tests on bigger number. of input tensors